src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -9,28 +9,22 @@
 using DelegateCommand = Dynamo.UI.Commands.DelegateCommand;
 using Dynamo.Models;
 using System.Windows.Data;
+using System.Globalization;
 
 namespace Dynamo.ViewModels
 {
-    public sealed class PathEnabledConverter : IValueConverter
+    public sealed class PathEnabledConverter : IMultiValueConverter
```

Comment:
not sure but this might be considered a breaking change ...
not even sure if anyone (API consumer) could have a use for this class ...

Reply:
it's a new class as of 2.12